src/main/resources/modules/contraceptives/female_sterilization.json

```diff
@@ -50,7 +50,8 @@
           "display": "Admission to surgical department"
         }
       ],
-      "direct_transition": "Tubal_Ligation_Procedure"
+      "direct_transition": "Tubal_Ligation_Procedure",
+      "reason": "History of Tubal Ligation"
```

Comment:
Remember the documentation of the encounter happens after it occurs. Also, right now the "reason" needs to be Condition. The reason cannot be a procedure.

Reply:
You could use something like ICD-10-CM Z30.9 "Encounter for contraceptive management, unspecified"

Reply:
I can't use ICD codes.

Reply:
The SNOMED equivalent is 315925004 "Contraceptive management (situation)" then map to ICD-10-CM in the BFD condition code map.

Reply:
> The reason cannot be a procedure.

Okay, I was wrong about that. It is usually a Condition, but it technically can be any `HealthRecord.Entry` assigned to an attribute.

Any mapping can be in the file, so that isn't a problem. I think the issue is that the "situation" has to already be in the record in order to use it in the module as the reason for the encounter. Let me try a few things...

Reply:
> The SNOMED equivalent is 315925004 "Contraceptive management (situation)" then map to ICD-10-CM in the BFD condition code map.

Where are you finding this? That concept does not exist, either by code or description, in US or International versions of SNOMED.

Reply:
I found the mapping in the UMLS MRCONSO file. It claimed to be from SNOMED-CT US edition but maybe its an old code that was since removed.

Reply:
How about SNOMED-CT 183996000 | Sterilization requested (situation)